src/Codeception/Module/DataFactory.php

```diff
@@ -0,0 +1,169 @@
+<?php
+namespace Codeception\Module;
+
+use Codeception\Lib\Interfaces\DataMapper;
+use Codeception\Lib\Interfaces\ORM;
+use Codeception\TestCase;
+use League\FactoryMuffin\FactoryMuffin;
+use League\FactoryMuffin\Stores\RepositoryStore;
+
+/**
+ * DataFactory allows you to easily generate and create test data using [**FactoryMuffin**](https://github.com/thephpleague/factory-muffin).
+ * DataFactory uses an ORM of your application to define, save and cleanup data. Thus, should be used with ORM or Frqmework modules.
+ *
+ * Generation rules can be defined in a factories file. You will need to create `factories.php` (it is recommended to store it in `_support` dir)
+ * Follow [FactoryMuffin documentation](https://github.com/thephpleague/factory-muffin) to set valid rules.
+ * Random data is set with [Faker](https://github.com/fzaninotto/Faker) library.
+ *
+ * ```php
+ * use League\FactoryMuffin\Faker\Facade as Faker;
```

Comment:
Faker support is provided by Factory Muffin Faker. If you want to enable faker support, then you need to add "league/factory-muffin-faker": "^1.0" too: